From c6ce0e9276a4b36e6c627cc733a59f775a253a12 Mon Sep 17 00:00:00 2001
From: Larry Finger <Larry.Finger@lwfinger.net>
Date: Wed, 8 May 2013 14:55:39 -0500
Subject: [PATCH 132/390] Remove /procfs entries to allow the driver to build
 on 3.10-rc0

This hack removes the entire set of procfs entries. They will ultimately
be added back in /sysfs.

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
---
 os_dep/os_intfs.c | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

--- a/drivers/staging/rtl8192du/os_dep/os_intfs.c
+++ b/drivers/staging/rtl8192du/os_dep/os_intfs.c
@@ -253,6 +253,17 @@ static int	rtw_proc_cnt = 0;
 
 #define RTW_PROC_NAME DRV_NAME
 
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(3, 8, 0))
+/* dummy routines */
+void rtw_proc_remove_one(struct net_device *dev)
+{
+}
+
+void rtw_proc_init_one(struct net_device *dev)
+{
+}
+
+#else	/* LINUX_VERSION_CODE > KERNEL_VERSION(3, 8. 0) */
 void rtw_proc_init_one(struct net_device *dev)
 {
 	struct proc_dir_entry *dir_dev = NULL;
@@ -657,7 +668,7 @@ void rtw_proc_remove_one(struct net_devi
 	}
 }
 #endif
-
+#endif
 uint loadparam(struct rtw_adapter *padapter, struct net_device *pnetdev)
 {
 
